HISTORY.md

```diff
@@ -2,4 +2,9 @@
 
```

Comment:
@jhaigh0 With the history file, you would typically put the new release information at the top - with the oldest release at the end of the file.